Source/ARTCrypto.h

```diff
@@ -28,13 +28,20 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 /**
- ARTCipherParams contains configuration options for a channel cipher, including algorithm, mode, key length and key.
- Ably client libraries currently support AES with CBC, PKCS#7 with a default key length of 256 bits. All implementations also support AES128.
+ The ARTCipherParams object contains properties to configure encryption for a channel.
  */
 @interface ARTCipherParams : NSObject <ARTCipherParamsCompatible>
+
+/// The algorithm to use for encryption. Only AES is supported.
```

Comment:
@tbedford @m-hulbert For consistency with the documentation for `mode`, should this not be the following?

```suggestion
/// The algorithm to use for encryption. Only `"AES"` is supported.
```

Reply:
This should just be `AES` (no " ") as per the canonical table.